tests/test_playbooks/inventory_plugin.yml

```diff
@@ -0,0 +1,162 @@
+---
+- hosts: container
+  vars:
+    foreman_groups:
+      - group_a
+      - group_b
+    foreman_hosts:
+      testhost1.example.com: group_a
+      testhost2.example.com: group_b
+  tasks:
+    - name: Skip tests on Ansible 2.8 and older
+      meta: end_play
+      when: ansible_version['full'] is version('2.9', '<')
+
+    - name: create database.yml
+      copy:
+        dest: /tmp/database.yml
+        content: |
+          development:
+            adapter: sqlite3
+            database: db/development.sqlite3
+            pool: 10
+            timeout: 5000
+          production:
+            adapter: sqlite3
+            database: db/production.sqlite3
+            pool: 10
+            timeout: 5000
+
+    - name: fetch Foreman container
+      docker_image:
+        name: quay.io/foreman/foreman:2.0-stable
+        source: pull
+
+    - name: start Foreman container
+      docker_container:
+        name: foreman
+        image: quay.io/foreman/foreman:2.0-stable
+        command: bundle exec bin/rails server -b 0.0.0.0
+        published_ports:
+          - "0.0.0.0:3000:3000"
+        volumes:
+          - "/tmp/database.yml:/home/foreman/config/database.yml"
+
+    - name: hack the gibson
+      command: "docker exec foreman sed -i 's/FactImporter.importer_for(type)/PuppetFactImporter/' /home/foreman/app/models/host/base.rb"
```

Comment:
this is a terrible hack, but for some reason `FactImporter.importer_for(type)` does return `nil` inside the container, whereas it should return `PuppetFactImporter` and I have no idea why